src/lualib/declarations/debug.d.ts

```diff
@@ -2,4 +2,5 @@
 
 declare namespace debug {
     function traceback(...args: any[]): string;
+    function getinfo(i: number, what?: string): any;
```

Comment:
This return value can be typed properly from https://www.lua.org/pil/23.1.html

Reply:
True, though in this case it would only partially help because the exact type depends on the input argument, so we would still rely on external assumptions.

Reply:
Yes, but if you type them all as optional properties it will already be a lot more readable/maintainable for other developers that might not know the details of this function.